qadi_multi_perspective.py

```
@@ -0,0 +1,284 @@
+        # Always show synthesis when multiple perspectives
+        if len(result.perspective_results) > 1:
+            display_synthesis(result)
+        
+        # Summary statistics
+        elapsed_time = time.time() - start_time
+        print(f"\n{'─' * 70}")
+        print(f"✅ Analysis completed in {elapsed_time:.1f}s")
+        print(f"💰 Total LLM cost: ${result.total_llm_cost:.4f}")
+        print(f"🔍 Perspectives used: {len(result.perspective_results)}")
+        
+    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)